variation/hgvs_dup_del_mode.py

```diff
@@ -84,24 +97,28 @@ def cnv_mode(self, ac: str, del_or_dup: str, location: Dict,
         if chromosome == 'X':
             copies = models.DefiniteRange(
                 min=0 if del_or_dup == 'del' else 2,
-                max=1 if del_or_dup == 'del' else 3
+                max=1 if del_or_dup == 'del' else 3,
+                type="DefiniteRange"
             )
         elif chromosome == 'Y':
             copies = models.Number(
-                value=0 if del_or_dup == 'del' else 2
+                value=0 if del_or_dup == 'del' else 2, type="Number"
```

Comment:
if these objects require explicit type parameters I think we should raise an issue in VRS Python about it because that... kind of sucks

Reply:
Ahh I forgot to make an actual issue for this. It was in my mind to look into. Will create an issue now